openedx/features/course_experience/static/course_experience/js/spec/LatestUpdate_spec.js

```diff
@@ -0,0 +1,23 @@
+/* globals $, loadFixtures */
+
+import 'jquery.cookie';
+import { LatestUpdate } from '../LatestUpdate';
+
+describe('LatestUpdate tests', () => {
+  describe('Test cookies', () => {
+    beforeEach(() => {
+      loadFixtures('course_experience/fixtures/latest-update-fragment.html');
+      new LatestUpdate({ messageContainer: '.update-message', dismissButton: '.dismiss-message button' }); // eslint-disable-line no-new
+    });
+
+    it('Test setting cookie', () => {
+      $('.dismiss-message button').click();
+      expect($('.update-message').attr('style')).toBe('display: none;');
+
+      // Check that the cookie still applies for a new latest update.
```

Comment:
I think I'd leave this out, because this is more a side-effect of our implementation than what I think is really wanted.  Ideally, the cookie would know the update that was dismissed, and not apply to a new update...but that goes beyond the scope of this work and is not part of the AC.

If you really wanted to test the cookie, you could possibly muck with the cookie and show that the update is displayed again, even without creating a new message.